errors.toml

```diff
@@ -1,187 +1,187 @@
 # AUTOGENERATED BY github.com/pingcap/errors/errdoc-gen
 # YOU CAN CHANGE THE 'description'/'workaround' FIELDS IF THEM ARE IMPROPER.
 
-["BR:Backup:ErrBackupChecksumMismatch"]
+["BRIE:Backup:ErrBackupChecksumMismatch"]
```

Comment:
🤔 are error codes supposed to be stable?

Reply:
For compatibility, I'll revert the change.